lib/asdf.sh

```diff
@@ -1,20 +1,20 @@
+# shellcheck shell=sh
```

Comment:
What is this giving us that the `scripts/shellcheck.bash` isn't already?

Reply:
Adding that to the top of the file allows editors with shellcheck plugins to know how to treat the file. These sort of comments also obviate the need to pass --shell to each invocation since it can be inferred from the comment.

Reply:
I would like to be consistent and since we aren't using this in the rest of the codebase to remove it. I would rather just be explicit in the `scripts/*` we use than spreading this information throughout the codebase.